specification.md

```diff
@@ -0,0 +1,344 @@
+# $signing_spec
+
+<table>
+<tr><td>$signing_spec
+<tr><td>A signature scheme for software supply chain metadata that avoids canonicalization
+<tr><td>2020-09-28
+<tr><td>Version 0.1.0
+</table>
+
+# Abstract
+
+This document proposes a new signature scheme for use by, among others, the
+in-toto and TUF projects. This signature scheme (a) avoids relying on
+canonicalization for security and (b) reduces the possibility of
+misinterpretation of the payload. The serialized payload is encoded as a string
+and verified by the recipient _before_ deserializing. A backwards compatible
+variant is available.
+
+# Specification
+
+$signing_spec does not rely on Canonical JSON or any other encoding /
+canonicalization. Instead, the producer records the signed bytes
+exactly as signed and the consumer verifies those exact bytes before parsing. In
+addition, the signature now includes an authenticated `payloadType` field
+indicating how to interpret the payload.
+
+```json
+{
+  "payload": "<Base64(SERIALIZED_BODY)>",
```

Comment:
💯  for [s/signed/payload](https://github.com/theupdateframework/tuf/blob/60dcb95656e8dca156e075dba68ed66715e94c30/tuf/api/metadata.py#L268-L270)!! :)